src/router/validation/ValidationUtilities.ts

```diff
@@ -0,0 +1,9 @@
+import { Validator } from 'fhir-works-on-aws-interface';
+
+export async function validateResource(validators: Validator[], resource: any) {
+    return Promise.all(
```

Comment:
We should run the validators in order. Otherwise the validation error for the same input may be non-deterministic.

Reply:
updated